test/pkg/environment/common/expectations.go

```diff
@@ -97,34 +97,6 @@ func (env *Environment) ExpectCreatedOrUpdated(objects ...client.Object) {
 	}
 }
 
-func (env *Environment) ExpectSettings() (res []v1.EnvVar) {
```

Comment:
These are just general cleanups? I'd prefer them to be broken out into a separate PR that I'd be happy to review and approve as to not clutter this one.